### DISCUSSION TITLE
Contig size viewer enhancements

### DISCUSSION THREAD

**Discussion by sjackman** [ablab/quast#170] [Ideas]
- Highlight misassembled contigs in red/orange.
- Show the positions of the breakpoints in the misassembled contigs.
- Display the "Contig info" sidebar.
- Include a link to the "Alignment view" of the selected contig.
- Optionally display the sizes of the aligned blocks rather than the unaligned contig sizes to visualize the NA50.

**Comment by vladsavelyev** [comment]
First 4 suggestions were implemented in v4.1.

The 5th item sounds very reasonable, but its implementation is relatively time consuming, so we are not including it in the upcoming release, but will include in one of the following releases. Keeping this issue to keep track of the 5th item.

Thank you Shaun for the great feedback!

**Comment by sjackman** [comment]
Thanks for implementing these features so quickly!
